src/main/java/uk/gov/hmcts/ccd/data/casedetails/search/MetaData.java

```diff
@@ -9,6 +9,8 @@
 
 public class MetaData {
 
+    public static final String JURISDICTION_PARAM = "jurisdiction";
```

Comment:
there's a bit of an issue here we have to address. 
This list of *_PARAM is used to keep track of metadata that is passed to the api as a query parameter. They are added to the ALL_METADATA map which is used to validate the request. 
Jurisdiction and case type were not there because they are not passed as query parameters. 

I think we should rename ALL_METADATA to METADATA_QUERY_PARAMETERS
and rename all the constants from *_PARAM to *_METADATA

Reply:
sounds good, will rename it.

Reply:
Done